Moblin/HaishinKit/Extension/Double+Extension.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+extension Double {
+    func nearestCommonFrameRate() -> Double {
```

Comment:
In my opinion it doesn't make sense to make this an extension of Double. The functionality is not related to the type, it just happens to use double.

Reply:
done